lab_1_classify_by_unigrams/main.py

```diff
@@ -49,6 +83,37 @@ def compare_profiles(
     :param profile_to_compare: a dictionary of a profile to compare the unknown profile to
     :return: the distance between the profiles
     """
+    if isinstance(unknown_profile, dict) and isinstance(profile_to_compare, dict) == True:
+        if ('name' and 'freq' in unknown_profile) and ('name' and 'freq' in profile_to_compare) == True:
+
+            predicted_tokens = profile_to_compare.get('freq')
+            actual_tokens = unknown_profile.get('freq')
+
+            for key in predicted_tokens.keys():
+                if key in actual_tokens == False:
+                    actual_tokens[key] = 0
+
+            for key in actual_tokens.keys():
+                if key in predicted_tokens == False:
+                    predicted_tokens[key] = 0
+
+            predicted = []
+            for value in predicted_tokens.values():
+                predicted.append(value)
+            predicted.sort()
```

Comment:
what are they sorted by? is it guaranteed that the frequencies in the predicted and actual lists correspond to the same letter at each position?

Reply:
It was a mistake. It would sort numbers itself, not the letters. Now I sort profiles at first and then work with already sorted alphabetically profiles, so I don't need to sort values again here.